src/__tests__/Search/__snapshots__/SearchResultsList.test.tsx.snap

```diff
@@ -354,6 +354,9 @@ exports[`SearchResultsList should match snapshot 1`] = `
                               </p>
                             </div>
                           </li>
+                          <span
+                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
+                          />
```

Comment:
I believe these changes come from the fact that we're `await`ing something after clicking.